packages/editor/src/components/block-drop-zone/index.js

```diff
@@ -101,6 +102,7 @@ class BlockDropZone extends Component {
 			return;
 		}
 
+		const dstIndex = dstClientId ? getBlockIndex( dstClientId, dstRootClientId ) : undefined;
```

Comment:
What `dst` stands for? It would read better if we would avoid such abbreviations.